src/modules/orders.ts

```diff
@@ -5,35 +5,48 @@ import { Order } from "../../generated/schema"
 
 export namespace orders {
 
-    export function invalidateOrder(orderId: string): void {
+    export function invalidateOrder(orderId: string, timestamp: BigInt): Order {
 
         let order = Order.load(orderId)
 
         if (!order) {
             order = new Order(orderId)
-            log.warning('Order {} was not found. It was created for being invalidated', [orderId])
+            log.info('Order {} was not found. It was created for being invalidated', [orderId])
```

Comment:
Nope, the preSig will enter by the preSig function at line 23 in the same file. preSig and trades both are using getOrCreateOrder helper, because both can asign an owner. This creation is different because it doesn't provide an owner. Makes sense?

Reply:
Got it!